mplc/scenario.py

```diff
@@ -805,22 +837,23 @@ def run(self):
         # -----------------
         # Preliminary steps
         # -----------------
-
-        self.plot_data_distribution()
+        if not self.save_folder is None:
```

Comment:
I think `if self.save_folder:` does the trick

Reply:
see above comment